test/index.test.js

```diff
@@ -65,3 +65,13 @@ test.serial('postMessage', async t => {
 	t.deepEqual(second.data[2], msg);
 	t.not(second.data[2], msg);
 });
+
+test.serial('close', async t => {
+	const worker = new Worker('./test/fixtures/close.mjs', { type: 'module' });
+	let closed = false;
```

Comment:
```suggestion
```

Reply:
Running this locally now and it needs an assert, it seems (I've never used ava before). Pushing that now.